src/convertToObject.js

```diff
@@ -6,7 +6,29 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (!sourceString.length) {
+    return {};
+  }
+
+  const noEmptyLines = sourceString
+    .split(';')
+    .map((line) => {
+      return line.trim();
+    })
+    .filter((item) => item.length > 0);
```

Comment:
According to the [CODE STYLE] checklist, when chaining methods, each method should be on a new line with the same indentation to improve readability. Consider formatting the chained methods `.split`, `.map`, and `.filter` to start on new lines with consistent indentation.